src/components/auth/torus/ready.js

```diff
@@ -0,0 +1,47 @@
+import retryImport from '../../../lib/utils/retryImport'
```

Comment:
does this used in torus auth only ? if it's used in the self custody too, better to move it to `src/components/auth/ready.js`